doc/user/polyglot.md

```diff
@@ -47,22 +47,23 @@ object.
 
 `Polyglot.import(name)` imports and returns a value with a given name.
 
-`Polyglot.import_method(name)` imports a method with a given name, and defines
-it in the top-level object.
+`Polyglot.import_method(name)` imports a value, which should be `IS_EXECUTABLE`,
+with a given name, and defines it in the top-level object.
 
 ## Using Ruby objects from a foreign language
 
-Using JavaScript as an example.
+Using JavaScript as an example - the left example is JavaScript, the right one
+is the corresponding action it takes on the Ruby object expressed in Ruby code.
 
-`obj[name/index]` calls `[name/index]` on the Ruby object.
+`object[name/index]` calls `object[name/index]`.
 
-`obj[name/index] = value` calls `[name/index] = value` on the Ruby object.
+`object[name/index] = value` calls `object[name/index] = value`.
```

Comment:
Should the JS syntax `object.name`/`object.name = value` also be shown here?
I think that's more idiomatic JS for reading/writing in the case name is an identifier.

Reply:
The internal wiki page for JS interop says to use `object[name/index]` exclusively. Discuss on Slack if you need to.